File: 2nd Year/1st Semester/MAP/Laborator/SEM1_LAB2_MAP/model/StackContainer.java
package sem1_2.model;

import java.util.ArrayList;
import java.util.List;

public class StackContainer extends AbstractCointainer {

    public StackContainer() {
        tasks = new Task[10];
        size = 0;
    }

    @Override
    public Task remove() {
        if (!isEmpty()) {
            return tasks[--this.size];
        }
        return null;
    }
}

File: 2nd Year/1st Semester/MAP/Laborator/Lab9/Lab9/model/StackContainer.cs
namespace Lab9.model;

public class StackContainer : AbstractContainer
{
    public StackContainer()
    {
        tasks = new Task[10];
        size = 0;
    }

    public override Task Remove()
    {
        if (!IsEmpty())
        {
            return tasks[--size];
        }

        return null;
    }
}